PyInstaller/__main__.py

```diff
@@ -113,8 +113,8 @@ def run(pyi_args=None, pyi_config=None):
 
         run_build(pyi_config, spec_file, **vars(args))
 
-    except KeyboardInterrupt:
-        raise SystemExit("Aborted by user request.")
+    except KeyboardInterrupt as e:
+        raise SystemExit("Aborted by user request.") from e
```

Comment:
Don't change `SystemExit` exceptions please. The change is pointless due to the way the internals of CPython work.